rack-protection/lib/rack/protection/cookie_tossing.rb

```diff
@@ -10,12 +10,24 @@ module Protection
     # Supported browsers:: all
     # More infos::         https://github.com/blog/1466-yummy-cookies-across-domains
     #
-    # Does not accept HTTP requests if the HTTP_COOKIE header contains more than one
-    # session cookie. This does not protect against a cookie overflow attack.
+    # Does not accept HTTP requests if the
```

Comment:
Since this references HTTP stuff, it seemed more logical to me to reference the HTTP header and not the CGI environment variable.  I don't feel strongly so happy to change back